Data/Sys/GameSettings/G2M.ini

```diff
@@ -16,6 +16,11 @@ CPUThread = False
 [Video_Settings]
 SafeTextureCacheColorSamples = 512
 
+# Using a widescreen heuristic modification
+# fixes the game changing aspect ratios during gameplay.
+
+WidescreenHeuristicTransitionThreshold = 20
```

Comment:
Altering the transition threshold seems like a workaround. We should be able to change the standard and widescreen ratio values. See #12164 and #12169.